scripts/tests/test-empty-report.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Are these test files for some automated testing that wasn't committed?  Or are they meant for manual testing?  They feel like something a reader wont know what to do with